presto-docs/src/main/sphinx/release/release-0.132.rst

```diff
@@ -5,4 +5,5 @@ Release 0.132
 General Changes
 ---------------
 
-* Change unit of Hive namenode latency JMX stats from seconds to milliseconds
+* Change unit of Hive metastore latency and namenode latency JMX stats from
+  seconds to milliseconds
```

Comment:
"Report metastore and namenode latency in milliseconds rather than seconds in JMX stats."